### PR TITLE
Speed up bootstrapping by building in parallel

### PR DESCRIPTION
**Pull request by bjorncs**

### CODE REVIEW

**Comment by baldersheim** [comment]
👍

File: bootstrap.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ else
 fi
 
 mvn_install() {
-    mvn --batch-mode install -Dmaven.test.skip=true -Dmaven.javadoc.skip=true "$@"
+    mvn --batch-mode --threads 1C -nsu install -Dmaven.test.skip=true -Dmaven.javadoc.skip=true "$@"
```
Comment (hakonhall) [inline bootstrap.sh]: 1.25C?

**Review by hakonhall — APPROVED** [review]

File: bootstrap.sh (hunk 1 above)
Comment (bjorncs) [inline bootstrap.sh]: Is that the optimal value? :)

Comment (hakonhall) [inline bootstrap.sh]: I have no idea - usually it's a bit over 1C and 25% over is fairly conservative. Consider this for later then.

**Pull request merged**